src/xcode/ENA/ENA/Source/Protocols/RequiresAppDependencies.swift

```diff
@@ -31,6 +31,7 @@ protocol RequiresAppDependencies {
 	var riskProvider: RiskProvider { get }
 	var exposureManager: ExposureManager { get }
 	var serverEnvironment: ServerEnvironment { get }
+	var warnOthers: OthersWarnable { get }
```

Comment:
As mentioned above, we should be careful not to put everything here that is not needed in a lot of places

Reply:
I fully agree. As mentioned above I will take this as refactoring in the next sprint